app/src/main/java/io/gnosis/safe/ui/safe/balances/SafeBalancesFragment.kt

```diff
@@ -47,5 +69,17 @@ class SafeBalancesFragment : SafeOverviewBaseFragment<FragmentSafeBalancesBindin
 
     override fun handleActiveSafe(safe: Safe?) {
         navHandler?.setSafeData(safe)
+        with(binding.balancesContent) {
+            adapter?.notifyItemChanged(currentItem)
+        }
     }
 }
+
+class BalancesPagerAdapter(fragment: Fragment) : FragmentStateAdapter(fragment) {
+
+    override fun getItemCount(): Int = 2
+
+    override fun createFragment(position: Int): Fragment =
```

Comment:
We don't have a collectible fragment yet (this is part of phase 2). I would only show one fragment (we want to make a public release before the collectibles feature)

Reply:
Hmm, that messes up a bit the whole navigation that @elgatovital prepared. If this is absolutely necessary, then I would create duplicate files with the suffix `tmp` otherwise this will really blow up in terms of complexity (because of the safe change updates and such)

Reply:
Why? Just add a <fragment> for now to the layout

Reply:
Or alternatively, could I just simple put an empty fragment under collectibles with a textview saying "Coming soon ... " or something like that? I realise that creating temp files will just make merging new feature into the graph almost impossible.

Reply:
I was afraid of the callbacks not working, but never mind, you were right. I just made everything `gone` (so that I don't have to get rid of the code for the viewBinding) and added the fragment